### ISSUE TITLE
Use shapes for default document labels

### ISSUE THREAD

**Issue by paulkater** [vkbo/novelWriter#2332]
As a colour blind person I find the difference between the Draft and Finished labels quite difficult to see:

![Image](https://github.com/user-attachments/assets/3e7fe562-7daf-462b-bc80-baf0e14384da)

For that reason I always change the Finished label to a colour I can easily distinguish:

![Image](https://github.com/user-attachments/assets/21db11ba-c68c-40a7-a1cc-8c91ee146498)

(Yes, I understand if many people think this is ugly but it helps me a lot.)

Would it be possible to make such a change global, i.e. change it inside novelWriter once so it is standard for all projects? I'd already be happy if this could be set/changed in some .ini or .config file by hand. vi is good for it and I can figure out the hex code for the colour without a problem.

I know my request may be a bit out of the ordinary as colour blindness isn't that common, but it's worth a question, right?

As always, tusen takk for a brilliant piece of software!

**Comment by vkbo** [comment]
Colour blindness is common enough, but even if it wasn't, accessibility is a good reason. I'm gradually working on making the app more accessible. When I started the project it was meant to be used by 2-3 people, so there is a lot of catching up to do on that front.

In any case, the icon shapes were added to assist with colour blindness or reduced colour vision in general. See #1786. The topic of setting global values was discussed too in #1847, and the problem is that they cannot actually be global with the current way they're implemented. See that issue for more details. But the solution was to instead add an import/export feature for them.

When you've set up all the status and importance labels you think you need, you can export them to a file each, and import them into every new project you create.

**Comment by paulkater** [comment]
W00T! Yes, I'm sure that will work. Shapes would already go a long way; that's also a clever addition. 😸 
And the import/export is a clever feature, I have completely overlooked that.
Thank you!
